src/ducks/transactions/TransactionModal.styl

```diff
@@ -30,13 +30,15 @@
     padding-left 24px
     padding-right 24px
     min-height 57px
-    cursor pointer
 
 .TransactionModalRow-disabled
     color coolGrey
     font-style italic
     cursor default
 
+.TransactionModalRow-clickable
+    cursor pointer
```

Comment:
It is so useful to be able to add cursor pointer quickly that I wonder if we should add a cursor utility class. Vote if you find it useful :D https://github.com/cozy/cozy-ui/issues/478